lib/phlex/heroicons/base.rb

```diff
@@ -3,10 +3,10 @@
 # rubocop:disable Lint/MissingSuper
 module Phlex
   module Heroicons
-    class Base < Phlex::View
+    class Base < Phlex::SVG
```

Comment:
I needed to inherit from `Phlex::SVG` so that the `path()` method would be found (and work) in the various icon files.